README.md

```diff
@@ -32,20 +46,24 @@ Becareful, some commands as no effect on iOS because Apple don't let us to do wh
   * c. Registering Wifi Network, will require user approval - and the network saved would not be controlled by the app (for deletion, updation, etc) [[docs](https://developer.android.com/guide/topics/connectivity/wifi-save-network-passpoint-config)];
 
 ## Access Point
 |                                       Description                                     |      Android       |         iOS          |
 | :------------------------------------------------------------------------------------ | :----------------: | :------------------: |
-| Getting the status of the Access Point (Disable, disabling, enable, enabling, failed) | :white_check_mark: |  :x:  |
-| Enabling / Disabling Access Point                                                     | :white_check_mark: |  :x:  |
-| Getting / Setting new credentials (SSID / Password)                                   | :white_check_mark: |  :x:  |
-| Enabling / Disabling the visibility of the SSID Access Point                          | :white_check_mark: |  :x:  |
+| Getting the status of the Access Point (Disable, disabling, enable, enabling, failed) | :warning:(1a) |  :x:  |
+| Enabling / Disabling Access Point                                                     | :white_check_mark:(1b) |  :x:  |
+| Getting / Setting new credentials (SSID / Password)                                   | :warning:(1a) |  :x:  |
+| Enabling / Disabling the visibility of the SSID Access Point                          | :warning:(1a) |  :x:  |
 | Getting the clients list (IP, BSSID, Device, Reachable)                               | :white_check_mark: |  :x:  |
 | Handling the MAC filtering                                                            | :sos: |  :x:  |
 
+:warning:(1): Wifi API changes in Android SDK >= 29, restricts certain behaviour:
+  * a. This has been deprecated and will always fail for >= 29 Android SDK.
+  * b. Uses [`startLocalOnlyHotspot` API](https://developer.android.com/reference/android/net/wifi/WifiManager#startLocalOnlyHotspot(android.net.wifi.WifiManager.LocalOnlyHotspotCallback,%20android.os.Handler)) to enable or disable WiFi AP. This can only be used to communicate between co-located devices connected to the created WiFi Hotspot. Note - (i) Enabling and Disabling WiFi AP needs to request location permission; (ii) The network created by this method will not have Internet access; (iii) There's no other way to set WiFi AP's SSID and Passphrase.
```

Comment:
Maybe just for completeness we should say that enabling and disabling is actually a _request_ to enable and disable as the LocalOnlyHotspot is shared (potentially) across applications and therefore a request to enable/disable may not not necessarily trigger the execution of it.

From the docs:

>  Each application can make a single request for the hotspot, but multiple applications could be requesting the hotspot at the same time. When multiple applications have successfully registered concurrently, they will be sharing the underlying hotspot. 
> ...
> Internally, requests will be tracked to prevent the hotspot from being torn down while apps are still using it. 
> ...
> Applications should be aware that the user may also stop the LocalOnlyHotspot through the Settings UI; it is not guaranteed to stay up as long as there is a requesting application.
> ...
> Applications should also be aware that this network will be shared with other applications